### PR TITLE
initial

### PR DESCRIPTION
**Pull request by Adman**
Signed-off-by: Adrian Matejov a.matejov@centrum.sk

### CODE REVIEW

**Comment by Adman** [comment]
to @mrshu 
cc @Krista @JakubNvk @pepol @tWido @Foxiy

File: dayinfo.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+from errbot import botcmd, BotPlugin
+from datetime import datetime, date
+from calendar import isleap
+
+from pynameday.slovakia import Slovakia as sknameday
+from workalendar.europe import Slovakia as skholiday
+
+
+DATE_FORMAT = '%d.%m.%Y'
```
Comment (pepol) [inline dayinfo.py:11]: I think this should be configurable.

File: dayinfo.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+
+from errbot import botcmd, BotPlugin
+from datetime import datetime, date
+from calendar import isleap
+
+from pynameday.slovakia import Slovakia as sknameday
+from workalendar.europe import Slovakia as skholiday
```
Comment (pepol) [inline dayinfo.py:8]: I feel like, in order to make this usable for more people, holidays (at least which calendar is to be used) should be configurable by users in per-channel config. What do you think @mrshu?

File: dayinfo.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        if len(args) == 1 and args[0] is not '':
+            today = validate_date(args[0])
+            if today is None:
+                return 'Not a valid date'
+
+        names = sknameday()
+        name = names.get_nameday(month=today.month, day=today.day)
+
+        if name is None:
+            return "Noone's got nameday"
```
Comment (mrshu) [inline dayinfo.py:98]: @Adman How about 'It's noone's nameday?' =)

File: dayinfo.py (hunk 2 above)
Comment (mrshu) [reply dayinfo.py:8]: @pepol you are of course right, but I do not know how would I personally go about doing that, other thank doing this in a bit hacky way. I am also not sure if this should be part of the first version -- note that this PR is a direct rewrite of a previous version.

**Pull request merged**